src/olympia/conf/prod/settings.py

```diff
@@ -40,7 +40,7 @@
 
 SERVICES_DATABASE = get_db_config('SERVICES_DATABASE_URL')
 
-SLAVE_DATABASES = ['slave']
+REPLICA_DATABASES = ['slave']
```

Comment:
@eviljeff do we also need to change `slave` to replica?

Reply:
same as in /dev/settings.py